build.gradle

```diff
@@ -9,6 +13,19 @@ allprojects {
     project.ext.set("publishGroupId", group)
 }
 
+nexusPublishing {
```

Comment:
A single publishing command is enough to publish all modules? (as I see we had this duplicated before)

Reply:
Yes, the script takes care of it 
You can see in the following link: https://mvnrepository.com/artifact/com.cloudinary that all modules got a dummy version

Reply:
I also uploaded a build to SonaType using this script and as you can see in the image below all modules are there
<img width="349" alt="image" src="https://user-images.githubusercontent.com/95848801/166432371-88159b52-c8cf-41fe-b492-b7d0da18af9f.png">